src/package-json/postInstall.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+const postInstallScriptName = 'bitBindings.js';
+
+type LinkProps = {
+  packagePath: string,
+  indexName: string,
+  fileContent: string
+};
+class LinkData {
+  packagePath: string;
+  indexName: string;
+  fileContent: string;
+
+  constructor( { packagePath, indexName, fileContent }: LinkProps) {
+    this.packagePath = packagePath;
+    this.indexName = indexName;
+    this.fileContent = fileContent;
+  }
+
+  toString() {
+    return JSON.stringify(this);
+  }
+}
+const scriptTemplate = (data) => `const fs = require('fs');
+const path = require('path');
+const arr = ${data}
+function mkdirp(filepath) {
+    var dirname = path.dirname(filepath);
+    if (!fs.existsSync(dirname)) {
+        mkdirp(dirname);
+    }
+    fs.mkdirSync(filepath);
+} 
+arr.forEach(linkMetaData => { 
+    mkdirp(path.join(__dirname,'node_modules', linkMetaData.packagePath));
+    const filePath = path.join(__dirname,'node_modules', linkMetaData.packagePath, linkMetaData.indexName);
+        fs.writeFileSync(filePath, linkMetaData.fileContent)
+})`;
+
+export default function generatePostInstallScript(writeDir: string, linkMeta: Array<Object>) {
```

Comment:
I guess the linkMeta should be Array<LinkProps>?
By the way i think it's better to use LinkProps[] and not Array<LinkProps> but i'm not sure about the diffrences